test/NuGet.Clients.Tests/NuGet.CommandLine.Test/UILanguageOverrideTests.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.CommandLine.Test
+{
+    public class UILanguageOverrideTests
+    {
+        [Theory]
+        [InlineData("en-US", "en-US")]
+        [InlineData("fr-FR", "fr-FR")]
+        [InlineData("pt-BR", "pt-BR")]
+        public void GetOverriddenUILanguage_WithEnvironmentVariableSet_ReturnsCultureInfo(string envvarValue, string cultureName)
```

Comment:
Since all the `InlineData` use the same value for both parameters, should we consolodate it down to 1 parameter?  Otherwise, can we add at least 1 test case where they have different values, to show that it's useful to have separate parameters?

Reply:
Removed second param. Thanks!